wordpress-mobile/WordPress-Android
# Deprecate wordpress blue and replace it with blue

Fixes #14026 

This PR replaces the `wordpress_blue` with the equivalent `blue` color based on the updated guidelines. This changes the `primary` color all over the app. 

To test:
- I can't see how this change could cause any problems or how exactly to test it. 

## Regression Notes
1. Potential unintended areas of impact
- none

2. What I did to test those areas of impact (or what existing automated tests I relied on)
- there are no areas of impact

3. What automated tests I added (or what prevented me from doing so)
- none (no reason to add automated tests)


PR submission checklist:
